cmd/promote/rhobs/rhobs_test.go

```
@@ -0,0 +1,245 @@
+	}
+
+	// Initialize git repo with app-interface remote
+	for _, args := range [][]string{
+		{"init", tmpDir},
+		{"-C", tmpDir, "remote", "add", "upstream", "git@gitlab.cee.redhat.com:service/app-interface.git"},
+		{"-C", tmpDir, "add", "."},
+	} {
+		cmd := exec.Command("git", args...)
+		cmd.Stdout = nil
+		cmd.Stderr = nil
+		cmd.Run()
```

golangci-lint (2.12.2)

[error] Error return value of `cmd.Run` is not checked

(errcheck)